src/lib/auth.js

```diff
@@ -38,14 +38,17 @@ export default authRouter;
  *       default:
  *         $ref: "#/components/responses/default"
  */
-authRouter.post('/login', async (req, res) => {
+authRouter.post('/', async (req, res) => {
   const { email, password } = req.body;
-  const sha3PWD = CryptoJS.SHA3(password);
```

Comment:
The route handler is async, there is no need to use `Promise.then` while can use async/await everywhere